openzwavemqtt/const.py

```diff
@@ -80,7 +81,10 @@ class ValueType(Enum):
     BYTE = "Byte"
     SHORT = "Short"
     INT = "Int"
-    BUTTON = "Button"
+    BITSET = "BitSet"
+    RAW = "Raw"
+    BUTTON = "Button"  # Currently not working in OZW
+    SCHEDULE = "Schedule"  # Currently unsupported by OZW
```

Comment:
FYI, these comments are not true. OZW supports these, just not as configuration parameters.